src/encord_active/lib/metrics/utils.py

```diff
@@ -55,12 +55,8 @@ def load_metric_dataframe(
     df = pd.read_csv(metric.path).sort_values([sorting_key, "identifier"], ascending=True).reset_index()
 
     if normalize:
-        min_val = metric.meta.get("min_value")
```

Comment:
My guess is that the intent with storing `min_value` and `max_value` in the metadata is to avoid recalculating this value each time you explore a metric in the UI. The difference should be noticed when the dataset is large enough.

Reply:
That makes sense, I'll add those to the StatsMetadata thing and make them available